src/output.rs

```diff
@@ -83,13 +83,15 @@ impl OutputType {
             let replace_arguments_to_less = pager.source == PagerSource::EnvVarPager;
 
             if args.is_empty() || replace_arguments_to_less {
-                p.arg("--RAW-CONTROL-CHARS");
+                if args.is_empty() || replace_arguments_to_less {
```

Comment:
Looks like you accidentally duplicated this if condition?

Reply:
Wow, that's embarrassing. Not sure how I missed it.